GetFacebookUserDetails

```diff
@@ -0,0 +1,62 @@
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.IO;
+using System.Net;
+//using System.Text;
+using System.Threading.Tasks;
+
+namespace GetFacebookUserDetails
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            
+                
+                y();
+            
+        }
+
+        static void y() {
+
+            WebClient fbaccess = new WebClient();
+            fbaccess.Proxy.Credentials = CredentialCache.DefaultNetworkCredentials;
+            var accesstoken = fbaccess.DownloadString("https://graph.facebook.com/oauth/access_token?grant_type=fb_exchange_token&client_id=1006289226156546&client_secret=c343d2c827f69c9c0895768a07ae0ad7&fb_exchange_token=EAACEdEose0cBAP7CYSje1E1AEMkQ9jhQbkozjWdjxgPRKjTAsJrWGr3s9PXIbY6IdnldffVD4QaJvcS1URkoXG0h5Sp6ca6pSA5PGFAsQKuB3dMplKiCXZBYJIrxpYbItjcC3ZBMEDZCKxKsX2LmCYNyi560Tx5g5J2K08WJAZDZD");
```

Comment:
@niharajay Clearly, this requires a `oauth/access_token` which certainly dosen't gel along with the missive of _Loklak_.

@mariobehling @Orbiter Closing this PR.